Cargo.toml

```diff
@@ -54,8 +54,6 @@ puffin = "0.18"
 raw-window-handle = "0.5.0"
 thiserror = "1.0.37"
 
-# Make the renderer `Sync` even on wasm32, because it makes the code simpler:
-wgpu = { version = "0.18.0", features = ["fragile-send-sync-non-atomic-wasm"] }
-
-# Use this to build wgpu with WebGL support on the Web *instead* of using WebGPU.
-#wgpu = { version = "0.18.0", features = ["webgl"] }
+wgpu = { git = "https://github.com/gfx-rs/wgpu", commit = "7774f31", features = [
+    "fragile-send-sync-non-atomic-wasm",
+] } # Make the renderer `Sync` even on wasm32, because it makes the code simpler:# TODO: Use a stable version of wgpu.
```

Comment:
```suggestion
    # Make the renderer `Sync` even on wasm32, because it makes the code simpler:
    "fragile-send-sync-non-atomic-wasm",
] } # TODO: Use a stable version of wgpu.
```